sklearn/ensemble/forest.py

```diff
@@ -94,19 +96,28 @@ def _generate_unsampled_indices(random_state, n_samples):
 
 
 def _parallel_build_trees(tree, forest, X, y, sample_weight, tree_idx, n_trees,
-                          verbose=0, class_weight=None):
+                          verbose=0, class_weight=None, max_samples=1.0):
```

Comment:
by `max_samples` I would naturally understand a total number, not a ratio... add explanation or `nb_samples`, `used_samples`?